### PR TITLE
Feature/sergeyro/tab remove inner css overrides - new prop

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4105590786

### CODE REVIEW

File: src/components/Tabs/__stories__/tabs.stories.mdx [hunk 1]
```diff
@@ -104,6 +104,7 @@ The width of the list is responsive to the screen's width.
 
 ### Motion
 
+// TODO this example demonstrates nothing - TabPanels animationDirection is getting overriden by TabsContext here, so no sense in passing animationDirection to TabPanels
```
Comment (SergeyRoyt) [inline src/components/Tabs/__stories__/tabs.stories.mdx:107]: Should we remove it? Or let's find our way around that?

File: src/components/Tabs/TabList/TabList.scss [hunk 2]
```diff
@@ -26,10 +26,10 @@
     .tabs-list {
       width: 100%;
 
-      .tab--wrapper {
+      .tab--wrapper-override {
```
Comment (orrgottlieb) [inline src/components/Tabs/TabList/TabList.scss:29]: why ?

Comment (SergeyRoyt) [reply src/components/Tabs/TabList/TabList.scss:29]: wanted to rename it somehow to show that it's styles which overrides `tab--wrapper` through Tab props, not through global class anymore
Can be renamed to `&_tab--wrapper` e.g. if u prefer. Or what r u proposing here?

Comment (orrgottlieb) [reply src/components/Tabs/TabList/TabList.scss:29]: oh, so this is the internal conflicts that we had ?

wrapper feels better

**Review by orrgottlieb — APPROVED** [review]

File: src/components/Tabs/__stories__/tabs.stories.mdx (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Tabs/__stories__/tabs.stories.mdx:107]: Not in this PR then...

**Pull request merged**